angular-client/src/components/landing-page/landing-page.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit } from '@angular/core';
+import { io } from 'socket.io-client';
+import { ImportantDataType } from 'src/enumeration/ImportantDataType';
+import APIService from 'src/services/api.service';
+import { SocketService } from 'src/services/socket.service';
+import Storage from 'src/services/storage.service';
+import { DataValue } from 'src/utils/socket.utils';
+
+@Component({
+  selector: 'landing-page',
+  styleUrls: ['./landing-page.component.css'],
+  templateUrl: './landing-page.component.html'
+})
+export default class LandingPage implements OnInit {
```

Comment:
is this page being in a components folder just an angular thing?

Reply:
uhh kind of but like everything in angular is a component so. What i did painfully find out is you have to put images under the assets folder or else it will just not be able to find it

Reply:
ah damn that's annoying LOL. I would say technically everything in React is a component as well, so if you could rename the 'components' folder to 'pages' and then make a separate 'components' page for re-useable components, but up to you. Either way there should eventually be some sort of organization of files once more stuff is added (inside of components if that's require or you end up keeping it)

Reply:
agreed